etc/dehydrated/hook.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+CONSUL_ROOT="http://localhost:8500/v1/kv/nginx"
```

Comment:
Absolutely. I'll do this. This PR is _very_ WIP, so some of what you see if just a result of me testing this out locally, outside of containers.